sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientBase.java

```diff
@@ -86,7 +86,7 @@ public abstract class IdentityClientBase {
     static final String WINDOWS_SWITCHER = "/c";
     static final String LINUX_MAC_SWITCHER = "-c";
     static final Pattern WINDOWS_PROCESS_ERROR_MESSAGE = Pattern.compile("'azd?' is not recognized");
-    static final Pattern SH_PROCESS_ERROR_MESSAGE = Pattern.compile("azd?: command not found");
+    static final Pattern SH_PROCESS_ERROR_MESSAGE = Pattern.compile("azd?:.*not found");
```

Comment:
little curious about why this didn´t show up in unit test...   I don't think `azd` is installed in the DevOps nodes...  so should we have seen an error for the test that checks when azd is not installed and we use the DefaultAzureCred?

Reply:
Our unit tests mock `authenticateWithAzureDeveloperCli` so we never actually hit this code path. :( (I just learned this also!) Improving Identity tests is a thing we need to focus on anyway - this is now on the list.